pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.2</version><!-- which version of Jenkins is this plugin built against? -->
+        <version>2.6</version>
```

Comment:
Just noticed that the minimum version still targets 2.6.  That works, but will force everyone to upgrade their Jenkins.  And there's a security fix in the 2.x line that has broken several plugins, so the adoption rate isn't that good.  Were you able to retest with a lower version of Jenkins than 2.6?